themes/default/content/docs/support/troubleshooting.md

```diff
@@ -481,17 +326,14 @@ const stackReference = new StackReference("...", { name: otherResource.outputVal
 const val = stackReference.getOutputSync("outputName");
 ```
 
-This warning may be benign. However, if you are experiencing crashes or hangs in Pulumi (especially in Node.js version 12.11.0 and
-above) and you see this warning, then it is likely that this is the source.
+This warning may be benign. However, if you are experiencing crashes or hangs in Pulumi (especially in Node.js version 12.11.0 and above) and you see this warning, then it is likely that this is the source.
```

Comment:
Same suggestion as above.

Reply:
@jkodroff i have no idea what _above_ means here? sry can you be more specific?